controllers/dashboard_test.go

```diff
@@ -163,6 +163,35 @@ var _ = Describe("DashboardController", func() {
 					Expect(len(matches)).To(BeZero())
 				})
 			})
+
+			Context("given `keyword` param is set", func() {
+				It("shows only `keyword` filter records", func() {
+					keywordFilter := "keyword_keyword_keyword"
+					user := FabricateUser(faker.Email(), faker.Password())
+					_ = FabricateKeyword(keywordFilter, faker.URL(), 0, user)
+					_ = FabricateKeyword(fmt.Sprintf("%v %v %v", faker.Word(), keywordFilter, faker.Word()), faker.URL(), 0, user)
+					_ = FabricateKeyword("paragraph_paragraph_paragraph", faker.URL(), 0, user)
```

Comment:
Again for explicitness, I recommend the following:

```suggestion
					_ = FabricateKeyword("nonexpected_keyword", faker.URL(), 0, user)
```